isis/src/galileo/objs/SsiCamera/SsiCamera.cpp

```diff
@@ -81,7 +81,7 @@ namespace Isis {
     // Get the start time in et
     PvlGroup inst = lab.findGroup("Instrument", Pvl::Traverse);
 
-    double et = iTime((QString)inst["StartTime"]).Et();
+    double et = getClockTime((QString)inst["SpacecraftClockStartCount"]).Et();
```

Comment:
@scsides Do you think it's reasonable to make this change? Do you know if `StartTime` is known to be more accurate for Galileo SSI or something like that?

Reply:
clock counts are nearly always more accurate or at least as accurate.